services/quotas-service/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/QuotasServiceDAL.java

```diff
@@ -86,7 +86,7 @@ public QuotaCheck checkQuotas() {
         // fargate
         serviceCode = "fargate";
         deployedCountMap.clear();
-        deployedCountMap.put("Fargate On-Demand resource count", getFargateResourceCount());
+        deployedCountMap.put("Fargate On-Demand vCPU resource count", getFargateResourceCount());
         quotasMap = getQuotas(serviceCode);
```

Comment:
Since Service Quotas is still returning the quota codes for the old non vCPU quotas, we should remove them from our list of things to look at.
```java
// Remove old on demand quota that have been replaced with the new vCPU quota
quotasMap.remove("Fargate On-Demand resource count");
quotasMap.remove("Fargate Spot resource count");
```

Reply:
I reflected what you suggested.